src/reachy2_sdk/parts/head.py

```diff
@@ -89,99 +89,92 @@ def __repr__(self) -> str:
     def neck(self) -> Orbita3d:
         return self._neck
 
-    def get_orientation(self) -> pyQuat:
-        """Get the current orientation of the head.
+    def get_current_state(
+        self, degrees: bool = True, round_int: Optional[int] = None, as_quat: bool = False
```

Comment:
not  big fan of `round_int`, at first I understood round as int. round isn't good enough?
also i'm not sure about the rounding in general. There are some in the __repr__ which are only there to make prints pretty.

Reply:
I had issues with the name round, in files where the function np.round() was import as round(). That's why I changed it